prism-highlighter.html

```diff
@@ -59,7 +59,12 @@
         return;
       }
 
+      if(event.detail._highlighted === true) {
```

Comment:
Sounds fair.

However if the listener functions are the same, addEventListener won't add it twice. example.

As prism-element doesn't hold any stateful properties, the implementation should avoid using bind in ready, which will prevent the scenario you just mentioned from happening.